packages/connect/src/protocol-connect/handler-factory.ts

```diff
@@ -153,6 +154,7 @@ function createUnaryHandler<I extends Message<I>, O extends Message<O>>(
     let status = uResponseOk.status;
     let body: Uint8Array;
     try {
+      if (opt.requireConnectProtocolHeader) requireProtocolVersion(req.header);
```

Comment:
probably just my OCD but I like curly braces or at least this to be on the same line.  I sorta wish we enforced https://eslint.org/docs/latest/rules/nonblock-statement-body-position